lib/project_types/extension/forms/questions/ask_app.rb

```diff
@@ -0,0 +1,53 @@
+module Extension
+  module Forms
+    module Questions
+      class AskApp
+        include ShopifyCli::MethodObject
+
+        property! :ctx
+        property :api_key
+        property! :prompt,
+          converts: :to_proc,
+          default: -> { CLI::UI::Prompt.method(:ask) }
+
+        def call(project_details)
+          project_details.tap(&method(:prompt_for_app))
+        end
+
+        private
+
+        def prompt_for_app(project_details)
+          project_details.app =
+            api_key.nil? ? choose_interactively(load_apps) : validate_app_ownership(api_key)
```

Comment:
I might have missed it but it looks like we're not prompting for the apiKey? Would the developer be expected to do `shopify create extension --apiKey=123` if they know the specific apiKey they want?

Reply:
Yes, if you'd like to set a particular API key, you have to pass it as a command line argument. Otherwise, we are providing them with a list of all of their apps and they can choose with the cursor keys.